.gitignore

```diff
@@ -1 +1,2 @@
 venv/
+.idea/
```

Comment:
don't think we should ignore .idea since it references python from venv see .idea/runConfigurations/entrypoint.xml